fix_subscription_now.py

```
@@ -0,0 +1,289 @@
+
+        if tier_result.data:
+            tier_id = tier_result.data[0]['id']
+            tier_name = tier_result.data[0]['name']
+            print(f"   Tier: {tier_name} ({tier_id})")
+        else:
+            # Default to pro_monthly
+            tier_id = 'pro_monthly'
+            print(f"   Tier: pro_monthly (default, couldn't match price ID)")
+
+        # Update user in database
+        print(f"\n[*] Updating database...")
```

Ruff (0.14.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)